src/tokens/saga.ts

```diff
@@ -133,26 +132,32 @@ export async function fetchTokenBalancesForAddress(
 ): Promise<FetchedTokenBalance[]> {
   const chainsToFetch = getSupportedNetworkIdsForTokenBalances()
   const userBalances = await Promise.all(
-    chainsToFetch.map((networkId) => {
-      return apolloClient.query<UserBalancesResponse, { address: string; networkId: string }>({
-        query: gql`
-          query FetchUserBalances($address: Address!, $networkId: NetworkId) {
-            userBalances(address: $address, networkId: $networkId) {
-              balances {
-                tokenId
-                tokenAddress
-                balance
+    chainsToFetch.map(async (networkId) => {
+      const response = await fetch(`${networkConfig.blockchainApiUrl}/graphql`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        },
+        body: JSON.stringify({
+          query: gql`
+            query FetchUserBalances($address: Address!, $networkId: NetworkId) {
+              userBalances(address: $address, networkId: $networkId) {
+                balances {
+                  tokenId
+                  tokenAddress
+                  balance
+                }
               }
             }
-          }
-        `,
-        variables: {
-          address,
-          networkId: networkId.replaceAll('-', '_'), // GraphQL does not support hyphens in enum values
-        },
-        fetchPolicy: 'network-only',
-        errorPolicy: 'all',
+          `,
+          variables: {
+            address,
+            networkId: networkId.replaceAll('-', '_'), // GraphQL does not support hyphens in enum values
+          },
+        }),
       })
+      return (await response.json()) as { data: UserBalancesResponse }
```

Comment:
do we want to check for the status code and throw if not 200? i think it doesn't make too much of a difference since the consumers are wrapping this with try/catch, just wanted to check if we've thought about it (since apollo always returns a 200)

Reply:
Good catch! I'll add this 👍